final_prep/mod_zero_hero.rb

```diff
@@ -55,30 +82,60 @@
 #   - citiesDestroyed (array)
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
-
+scary_monster = {name: "Heinrich the Brave", smell: "grotesque", weight: 220,
```

Comment:
No need to change here, but for future reference, hashes are a little easier to read when each key/value pair is formatted to be on its own line. Like so:
```
scary_monster = {
    name: "Heinrich the Brave",
    smell: "grotesque",
    weight: 220,
     ....
}
```